README.md

```diff
@@ -1,73 +1,22 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
-
-> Description the project.
-
+# Hello-microverse
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+html and css
```

Comment:
- Please, kindly remove this from here as you've updated it in the previous review as I've required the changes.